foia_hub/views.py

```diff
@@ -91,25 +100,14 @@ def request_form(request, slug=None):
         resource = AgencyResource()
 
     data = resource.detail(slug).value
-    template = env.get_template('request/form.html')
-    return HttpResponse(template.render(profile=data, slug=slug))
+    return render(
+        request,
+        'request/form.html',
+        {'profile': data, 'slug': slug})
 
 
-def request_success(request, id):
-    #   @todo: this makes it easy for an attacker to harvest email addresses
-    #   -- just look at all of the /success/##s in sequential order
-    foia_request = get_object_or_404(FOIARequest, pk=id)
-    requester = foia_request.requester
-    office = foia_request.office
-    agency = foia_request.agency or office.agency
-
-    template = env.get_template('request/success.html')
-    return HttpResponse(template.render(
-        foia_request=foia_request, requester=requester, office=office,
-        agency=agency))
-
-
-def get_domain(url):
-    return "%s/..." % urlparse(url).netloc
-
-env.filters['get_domain'] = get_domain
+def request_noop(request):
+    """ We have a request form that does nothing. Let's ensure the user knows
+    that in the slim chance the form gets turned on in an environment it
+    shouldn't be on in. """
+    return render(request, 'request/noop.html', {})
```

Comment:
404 feels more error like to me. This is a placeholder. It's not an ideal situation. So I use this to provide a little bit more information than a 404 page would.